### PR TITLE
Create .htaccess for new topologicpy ontology

### PR DESCRIPTION
**Pull request by wassimj**
We have created a new ontology for 3D Spatial Design and Analysis called topologicpy. This is a redirect for it.

### CODE REVIEW

**Comment by dgarijo** [comment]
Please add a readme file indicating who the maintainer for this w3id is. See https://github.com/perma-id/w3id.org?tab=readme-ov-file#creating-a-new-identifier

**Comment by wassimj** [comment]
@djodjoni Done. Apologies for the oversight. Thank you.

**Comment by dgarijo** [comment]
@wassimj the readme should be a little simpler. 
Please add just the overview and the contact details of the maintainer, including your github id. Its purpose is to help us know if someone making changes to this w3id in the future is in the maintainer list.

**Comment by wassimj** [comment]
@djodjoni Sorry about that. I hope this simplified README is fit-fo-purpose. Thanks again.

**Comment by dgarijo** [comment]
Thanks!

File: topologicpy/README.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+* hasClusterMember: Links a cluster to its members.
+* hasFreeCells: Links a cluster to its free cells.
+* hasFreeShells: Links a cluster to its free shells.
+* hasFreeFaces: Links a cluster to its free faces.
+* hasFreeWires: Links a cluster to its free wires.
+* hasFreeEdges: Links a cluster to its free edges.
+* hasFreeVertices: Links a cluster to its free vertices.
+* hasKeys: Links a dictionary to its keys.
+* hasValues: Links a dictionary to its values.
+* Data Properties
```
Comment (TallTed) [inline topologicpy/README.md]:
```suggestion

## Data Properties

```

File: topologicpy/README.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+* Grid: Represents a spatial structure dividing space into regular intervals.
+* Dictionary: Represents a key-value store for metadata.
+* Aperture: Represents an element indicating an opening or hole.
+* Context: Represents the environment or settings in which objects exist.
+* Vector: Represents a mathematical entity with magnitude and direction.
+* Matrix: Represents a rectangular array of numbers used for transformations.
+* Graph: Represents a collection of nodes and edges.
+
+## Object Properties
+* hasDictionary: Links a topology object to its dictionary.
```
Comment (TallTed) [inline topologicpy/README.md]:
```suggestion
## Object Properties

* hasDictionary: Links a topology object to its dictionary.
```

**Review by TallTed — COMMENTED** [review]
A little markdown repair...

**Pull request merged**